### ISSUE TITLE
Not using proxy

### ISSUE THREAD

**Issue by ghost** [Merubokkusu/Discord-S.C.U.M#308]
I noticed that my tokens get banned quickly. After that, I realized that proxy traffic was not wasted. For some reason, no matter what kind of proxy I use, it does not use it. I looked through the debugger, all the values ​​are assigned to the necessary variables, but the traffic is not wasted. What could be the problem?

**Comment by OrionCodeDev** [comment]
same here

**Comment by OrionCodeDev** [comment]
Any update? @Merubokkusu

**Comment by TortureGG** [comment]
Yeah, everything works fine without proxy, but not working with free proxy, got this response when launching gateway "bot.gateway.run()"

got this response  (sometimes with 403 error status, sometimes 400 or 405)

[gateway] < failed CONNECT via proxy status: 405
[gateway] websocket closed
[gateway] Connection Dropped. Retrying in 10 seconds.
